systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaUtils.java

```diff
@@ -330,9 +330,9 @@ public static boolean verifyPodDynamicConfiguration(String kafkaPodNamePrefix, S
 
         Map<String, ConfigModel> configs = KafkaUtils.readConfigModel(kafkaVersion);
 
-        LOGGER.info("This is configs {}", configs.toString());
+        LOGGER.debug("Kafka config {}", configs.toString());
```

Comment:
@see-quick is this log on INFO level worthy in case of some debug?

Reply:
reverting back to info.